x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/700_rrf_retriever_search_api_compatibility.yml

```diff
@@ -960,3 +980,87 @@ setup:
   - length: { hits.hits : 1 }
 
   - match: { hits.hits.0._id: "1" }
+
+---
+"rrf retriever with inner_hits for sub-retriever":
```

Comment:
Should we skip the BWC tests here?

Reply:
Added a new capability in https://github.com/elastic/elasticsearch/pull/116408/commits/1172be3fd1d2463a784157b3f2dcca0fd7e12929